ontology/yaml/resources/HVAC/entity_types/VLV.yaml

```diff
@@ -58,4 +58,9 @@ VLV_CDWPVM:
   - VLV
   - CDWPVM
 
-  
\ No newline at end of file
+VLV_SWCM:
+  description: "Solenoid water valve control and monitoring."
+  is_canonical: true
+  implements:
+  - VLV
+  - SWCM
```

Comment:
what kind of valve is it? what system does it live on? perhaps one of the existing types covers this better? i am concerned with using terms like `run_command` and `run_status` here. If we need more generic terms for the valve command, i suggest `valve_command` and `valve_status`, even though it deviates slightly from our other conventions.

Reply:
Hi sir,
the valve (solenoid water valve) is independent and will be controlled through BMS. I'll use yuor suggestions, thanks.